sdlib/d/gc/tcache.d

```diff
@@ -806,3 +841,27 @@ unittest arraySpill {
 	testSpill(16384, [0, 1, 2, 500, 16000, 16383, 16384]);
 	testSpill(20480, [0, 1, 2, 500, 20000, 20479, 20480]);
 }
+
+unittest finalization {
+	// Faux destructor which simply records most recent kill:
+	static size_t lastKilledUsedCapacity = 0;
+	static void* lastKilledAddress;
+	static uint destroyCount = 0;
+	static void destruct(void* body, size_t size) {
```

Comment:
body used to be a keyword, and it poorly highlighted. ptr is better, this what it is everywhere else anyway.